src/contractors.jl

```diff
@@ -72,20 +69,69 @@ function 𝒩{T}(f, f′, X::Interval{T})
 end
 
 
-IntervalArithmetic.mid(X::IntervalBox, α) = mid.(X, α)
-
 doc"""
 Multi-variable Newton operator.
 """
 function 𝒩(f::Function, jacobian::Function, X::IntervalBox)  # multidimensional Newton operator
-
     m = IntervalBox(Interval.(mid(X, where_bisect)))
```

Comment:
I think we can get rid of `where_bisect`, since calling `mid` without the second argument now does this automatically.

Reply:
Ah yes, you're totally right, sorry!

Reply:
Ouch that division is nasty...

Reply:
Not sure what that should give.

Reply:
In fact we have

```julia
0/X == [0, 0]
```

for any `X` that contains 0.

A quick look at the IEEE standard did not help much (I may read that in full someday to really understand all the stuff about flavors and decorated intervals). Note that `extended_div` suffers the same problem for now

```julia
julia> extended_div(0..0, -1..1)
([0, 0], ∅)
```

but here the standard  definition (10.5.5 Two output division) indicates that it should return `([-∞, ∞], ∅)`.

Reply:
Yes, I agree that `extended_div` is wrong here, according to the standard and comparing with the Octave interval arithmetic package; cc @eeshan9815 .

On the other hand, for standard division the situation is not so clear; the Octave package also returns `[0, 0]` in this case.